src/main/java/com/cloudbees/jenkins/plugins/bitbucket/api/BitbucketBuildStatus.java

```diff
@@ -30,6 +30,13 @@
 
 public class BitbucketBuildStatus {
 
+    // indicates that a build for the commit completed successfully
```

Comment:
Could I propose enumeration to handle a set of constants value? You can also change the field type since Jackson should handle it in the right way.

Reply:
Hi.
Please, check if I understood you correctly.
Thanks.